workspaces/global-header/plugins/global-header/src/components/HeaderDropdownComponent/ProfileDropdown.tsx

```diff
@@ -94,17 +81,24 @@ export const ProfileDropdown = ({ layout }: ProfileDropdownProps) => {
     <HeaderDropdownComponent
       buttonContent={
         <Box sx={{ display: 'flex', alignItems: 'center', ...layout }}>
-          <AccountCircleOutlinedIcon fontSize="small" sx={{ mr: 1 }} />
-          <Typography
-            variant="body2"
-            sx={{
-              display: { xs: 'none', md: 'block' },
-              fontWeight: 500,
-              mr: '1rem',
-            }}
-          >
-            {user?.displayName ?? 'Guest'}
-          </Typography>
+          {backstageIdentity && (
+            <>
+              <AccountCircleOutlinedIcon fontSize="small" sx={{ mr: 1 }} />
+              <Typography
+                variant="body2"
+                sx={{
+                  display: { xs: 'none', md: 'block' },
+                  fontWeight: 500,
+                  mr: '1rem',
+                }}
+              >
+                <EntityDisplayName
+                  entityRef={backstageIdentity.userEntityRef}
+                  hideIcon
```

Comment:
```suggestion
                  hideIcon
                  disableTooltip
```

But please take a look into the comment above, maybe we should use directly displayName and can remove the EntityDisplayName here? Wdyt?

But when it has a benefit that I don't see, let us disable the tooltip, or?